.eslintrc.json

```diff
@@ -53,7 +52,14 @@
                 "ignoreRestSiblings": true
             }
         ],
-        "@typescript-eslint/interface-name-prefix": ["error", { "prefixWithI": "never" }]
+        "@typescript-eslint/naming-convention": ["error", {
```

Comment:
Not that I know. I only replaced the existing rule `@typescript-eslint/interface-name-prefix` because [it was removed](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/naming-convention.md#enforce-that-interface-names-do-not-begin-with-an-i).

Reply:
this rule is already defined in the base config we use. I think we can remove it from here.